backend/tests/test_document_class.py

```diff
@@ -1,5 +1,33 @@
 from api.models import db, DocumentClass, Document
 
+import requests
+
+BACKEND_URL = "http://localhost:8000/"
```

Comment:
Can you change the BACKEND_URL to the deployed version?
https://h4i-infra-server.danielwonchoi.now.sh/

Reply:
done